includes/derivatives.inc

```diff
@@ -193,12 +197,12 @@ function islandora_scholar_add_preview_derivative(AbstractObject $object, $force
  *
  * @see hook_islandora_derivative()
  */
-function islandora_scholar_add_jpg_derivative(AbstractObject $object, $file_uri, $dsid, $width, $height, $force, $source_dsid) {
+function islandora_scholar_add_jpg_derivative(AbstractObject $object, $file_uri, $dsid, $width, $height, $colorspace, $force, $source_dsid) {
```

Comment:
Should we be changing the function signature here in case someone is calling this in their own module? 

```php
function islandora_scholar_add_jpg_derivative(AbstractObject $object, $file_uri, $dsid, $width, $height, $force, $source_dsid, $colorspace = NULL)
```

Might be more appropriate.

Reply:
Definitely agree! Also caught some missing stuff. Thanks for the feedback 👍

Reply:
+1 to @jonathangreen's suggestion.